perf.cmd

```diff
@@ -0,0 +1,10 @@
+@echo off
```

Comment:
You can use this to run perf tests.  I suspect @dpodder was looking for ways to make things simple, but you should know that in the end, all you really need to do is dotnet run the PerfHarness exe, which will do all of the work, including pulling all of the dependencies required to actually run.  No more having to cherry-pick dependencies into a single directory to run.